test/statsd/statsd_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+
+package statsd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/environment"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
+)
+
+const namespace = "StatsD"
+
+type StatsDTestSuite struct {
```

Comment:
I still don't really understand why we need to add a new test suite struct

Reply:
so we can test statsd only in this suite. I should probably move existing statsd testing out of metrics_value_benchmark into this suite.

Reply:
We talked about this on slack. I'm okay with this for now. Eventually we will want to revisit how all of these tests are structured because it is unruly. #218 covers this